### PR TITLE
Practicing GitHub Flow

### PR DESCRIPTION
**Pull request by freebong3**
## Test

- [x] 1

- [ ] 2

### CODE REVIEW

**Comment by freebong3** [comment]
:+1: :+1:

**Comment by eitah** [comment]
:8ball:

**Comment by freebong3** [comment]
please comment on #3

File: jhkim_bio.md [hunk 1]
```diff
@@ -0,0 +1 @@
+demo test
```
Comment (pscutts) [inline jhkim_bio.md]: this is interesting

**Pull request merged**